DataFormats/Scouting/interface/Run3ScoutingPhoton.h

```diff
@@ -71,6 +73,7 @@ class Run3ScoutingPhoton {
   float sMaj() const { return sMaj_; }
   float seedId() const { return seedId_; }
```

Comment:
Unrelated to this PR, but please consider addressing it: this looks wrong, b/c `seedId_` is an `unsigned int`.